### PR TITLE
Eisa dev

### PR DESCRIPTION
**Pull request by eisascience**

### CODE REVIEW

File: R/Seurat_III.R [hunk 1]
```diff
@@ -226,19 +226,27 @@ includeCellCycleGenes = T){
     }
 
     if (alignData && length(seuratObjs) > 1) {
-      if (!HasStepRun(so, 'NormalizeData', forceReCalc = forceReCalc)) {
```
Comment (bbimber) [inline R/Seurat_III.R:229]: @eisascience please be aware, this is changing from Dev.  we want to keep these changes: forceReCalc = forceReCalc

File: R/Seurat_III.R [hunk 2]
```diff
@@ -226,19 +226,27 @@ includeCellCycleGenes = T){
...
+      if (!HasStepRun(so, 'NormalizeData')) {
         print('Normalizing')
         so <- NormalizeData(object = so, verbose = F)
       } else {
         print('Normalization performed')
       }
 
-      if (!HasStepRun(so, 'FindVariableFeatures', forceReCalc = forceReCalc)) {
+      if (!HasStepRun(so, 'FindVariableFeatures')) {
```
Comment (bbimber) [inline R/Seurat_III.R:236]: same as above

**Review by bbimber — CHANGES_REQUESTED** [review]

File: R/Seurat_III.R (hunk 1 above)
Comment (eisascience) [reply R/Seurat_III.R:229]: I agree but, in this function there was no forceReCalc defined, nor did it make sense to have. I will double check.

Comment (eisascience) [reply R/Seurat_III.R:229]: We are talking about MergeSeuratObjs() function so, not sure why we want forceReCalc in there.

File: R/Seurat_III.R (hunk 2 above)
Comment (bbimber) [reply R/Seurat_III.R:236]: sorry, you're absolutely right.  

in practice, MergeSeuratObjs is most likely to be called on raw, unprocessed data, so it's unlikely this has been performed anyway.  this method is currently behaving as though it has forceReCalc=F, with no ability for the user to change that.  I think this is OK.

File: R/Seurat_III.R (hunk 1 above)
Comment (bbimber) [reply R/Seurat_III.R:229]: yes, you're right.  see reply to other comment

**Review by bbimber — APPROVED** [review]

**Pull request merged**